src/content/en/tools/chrome-devtools/network-performance/index.md

```diff
@@ -0,0 +1,257 @@
+project_path: /web/_project.yaml
+book_path: /web/tools/_book.yaml
+description: Get started analyzing network performance.
+
+{# wf_updated_on: 2016-12-15 #}
+{# wf_published_on: 2016-12-15 #}
+
+<style>
+.devtools-inline {
+  max-height: 1em;
+  vertical-align: middle;
+}
+figcaption {
+  text-align: center;
+}
+</style>
+
+# Get Started with Analyzing Network Performance in Chrome DevTools {: .page-title }
+
+{% include "web/_shared/contributors/kaycebasques.html" %}
+
+In this step-by-step, interactive tutorial, you learn how to use the Chrome
+DevTools Network panel to understand why a particular web page is loading
+slowly. The general workflow that you learn is helpful for analyzing network
+performance on any web page.
+
+## Step 1: Set up DevTools {: #set-up }
+
+Suppose that you're receiving reports from mobile users that a particular web
+page is slow. Your job is to make the page fast.
+
+Follow along with the instructions below to open DevTools on the page that
+your users are reporting is slow.
+
+1. Click **Open Slow Page**. The page opens in a new tab.
+
+     <a href="https://googlechrome.github.io/devtools-samples/network/gs/v1.html"
+       target="devtools" class="gc-analytics-event" rel="noopener noreferrer"
+       data-category="DevTools / Network / Get Started"
+       data-label="Slow Page Opened">
+       <button>Open Slow Page</button>
+     </a>
+
+1. While the page is in focus, press
+   <kbd>Command</kbd>+<kbd>Shift</kbd>+<kbd>I</kbd> (Mac) or
+   <kbd>Control</kbd>+<kbd>Shift</kbd>+<kbd>I</kbd> (Windows, Linux) to
+   open DevTools on the page.
+
+1. In DevTools, click the **Network** tab.
+
+     <figure>
+       <img src="imgs/get-started-network-panel.png"
+         alt="The Chrome DevTools Network panel, opened on the slow
+              page that you're going to diagnose.">
+       <figcaption>
+         <b>Figure 1</b>. The Chrome DevTools Network panel, opened on the slow
+         page that you're going to diagnose.
+       </figcaption>
+     </figure>
+
+     <aside class="note">
+       <b>Note:</b> For the rest of the screenshots, DevTools is undocked to a
+       separate window, so that you can see its contents better.
+     </aside>
+
+1. Enable **Capture Screenshots** ![Capture
+   Screenshots][screenshots]{:.devtools-inline}. It turns blue when it's
+   enabled. While DevTools reloads the page, it captures screenshots so
+   that you can see how the page load progresses over time.
+
+## Step 2: Emulate a mobile user's experience {: #emulate }
+
+When you test the page on your laptop or desktop, the network performance
+that you see can deceive you. Your browser is probably caching resources from
```

Comment:
Work on active voice a bit more in this paragraph, and leaning the language. Get this intro as short and active and spritely as possible. It's great content-- just make it smooth like butter. I'll give a little hint on this one-- just so you can see, but for sure, even my attempt can be leaner!

Testing network performance on a laptop or desktop can be deceiving. Your internet connect is must faster than on mobile and the browser caches resources from previous visits to the page.  Simulate mobile user experience using **Disable Cache** and **Network Throttling**.